test/unit/dropin.js

```diff
@@ -1400,7 +1400,10 @@ describe('Dropin', () => {
           verify: jest.fn().mockResolvedValue({
             nonce: 'new-nonce',
             liabilityShifted: true,
-            liabilityShiftPossible: true
+            liabilityShiftPossible: true,
+            threeDSecureInfo: {
+              threeDSecureAuthenticationId: 'id'
```

Comment:
In the reference docs (https://braintree.github.io/braintree-web/3.63.0/ThreeDSecure.html#~verifyPayload), the only properties listed on the `threeDSecureInfo` are `liabilityShiftPossible` and `liabilityShifted`. Why do we have the property `threeDSecureAuthenticationId` here?

Reply:
Good callout!

We mention this property specifically in the developer docs: (see for it in here: https://developers.braintreepayments.com/guides/3d-secure/client-side/javascript/v3#drop-in-ui )

I've asked our fraud tools team what things we should be exposing in addition to those 2 in the client reference docs. We'll hotfix the docs for this version when I get an answer.